hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/INodeFile.java

```diff
@@ -728,6 +732,15 @@ public void clearBlocks() {
     this.blocks = BlockInfo.EMPTY_ARRAY;
   }
 
+  /**
+   * This method replaces blocks in a file with the supplied blocks. Make sure
+   * you know what you are doing when you are calling this!
```

Comment:
We can remove this sentence: `Make sure you know what you are doing when you are calling this!`. I think this applies to most NameNode code. 🙂